zephyr/Kconfig

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2020 Legrand North America, LLC.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Kconfig to configure Zephyr module ZXA
+
+menuconfig ZXA
+	bool "ZXA (Zephyr eXample Module)"
```

Comment:
Wouldn't this be ZXM, not ZXA?

Or Zephyr eXAmple module, not Zephyr eXample Module.